app/views/corrections/CorrectionReturnPeriodView.scala.html

```diff
@@ -26,7 +26,7 @@
 
 @layout(pageTitle = title(form, messages("correctionReturnPeriod.title"))) {
 
-    @formHelper(action = controllers.corrections.routes.CorrectionReturnPeriodController.onSubmit(mode, period), 'autoComplete -> "off") {
+    @formHelper(action = controllers.corrections.routes.CorrectionReturnPeriodController.onSubmit(mode, period, Index(0)), 'autoComplete -> "off") {
```

Comment:
Should the index be passed as a parameter?

Reply:
yes, thanks for spotting that, done now